src/styles/main.scss

```diff
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import "header", "navigation", "container", "card", "reset";
```

Comment:
"reset" import should be in the 1st place here because it contains global stylings rules. When using an @import rule remember that allows you to import a style sheet into another style sheet.